README.md

```diff
@@ -37,9 +37,9 @@ Specific to Windows:
 
 ### macOS
 
-Follow the [React Native CLI Quickstart](https://reactnative.dev/docs/environment-setup?guide=native&os=macos&platform=android) instructions for macOS.
+Follow the [React Native Environment Setup](https://reactnative.dev/docs/set-up-your-environment) instructions for macOS with Android as well as iOS target OS.
```

Comment:
Changed the url here since I found the React Native's "Set up your environment" guide easier than Expo's guide. Accordingly, I changed the guide title over here to avoid any confusion.